doc/content/integrations/data-formats/_index.md

```diff
@@ -116,8 +116,8 @@ The JSON uplink messages use the following format:
       "data_rate_index": 5,                  // LoRaWAN data rate index
       "coding_rate": "4/6",                  // LoRa coding rate
       "frequency": "868300000",              // Frequency (Hz)
-      "timestamp": 2463457000,               // Timestamp of the gateway concentrator when the message has been received
-      "time": "2020-02-12T15:15:45.787Z"     // ISO 8601 UTC timestamp at which the uplink has been received by the gateway
+      "timestamp": 2463457000,               // Timestamp of the gateway concentrator when the accepted uplink message has been received
+      "time": "2020-02-12T15:15:45.787Z"     // ISO 8601 UTC timestamp at which the accepted uplink has been received by the gateway
```

Comment:
Actually I think we should reset these fields here so they disappear, @rvolosatovs what do you think?

If not, it's not really "accepted" but "deduplicated", and I'm afraid it's pretty random which one is chosen here

Reply:
Reset them where?

Reply:
In the Network Server

Reply:
Network Server discards these, so these are only seen and used by Gateway Server.

Reply:
> Network Server does not use these, so they are only used by Gateway Server.

Gateway Server sets them as part of the individual `RxMetadata`. It remains useful in there.

> I think NS should unset them as well, in fact.

That's what I mean. It doesn't make sense anymore to have this information _after_ deduplication in the `Settings`; but they should stay in the settings

Reply:
@benolayinka remove these two fields here for now as we'll get rid of this

Please also file issue for unsetting these field values in NS after deduplication